stackset_template.tf

```diff
@@ -0,0 +1,88 @@
+    },
+    "Resources": {
+        "IAMR7FYC": {
+            "Type": "AWS::IAM::Role",
+            "Properties": {
+                "ManagedPolicyArns": [
+                    {
+                        "Ref": "IAMMP12SQ7"
+                    }
+                ],
+                "RoleName": "ExpelRole",
+                "AssumeRolePolicyDocument": {
+                    "Version": "2012-10-17",
+                    "Statement": [
+                        {
+                            "Effect": "Allow",
+                            "Principal": {
+                                "AWS": { "Ref": "ExpelAssumeRoleARN" }
+                            },
+                            "Action": "sts:AssumeRole",
+                            "Condition": {
+                                "StringEquals": {
+                                    "sts:ExternalId": { "Ref": "ExpelCustomerOrganizationGUID" }
+                                }
+                            }
+                        }
+                    ]
+                }
+            }
+        },
+        "IAMMP12SQ7": {
+            "Type": "AWS::IAM::ManagedPolicy",
+            "Properties": {
+                "ManagedPolicyName": "ExpelAccessPolicy",
+                "PolicyDocument": {
+                    "Version": "2012-10-17",
+                    "Statement": [
+                        {
+                            "Effect": "Allow",
+                            "Action": [
+                                "ec2:DescribeInstances",
+                                "ec2:DescribeRegions",
+                                "ec2:DescribeSecurityGroups",
+                                "iam:List*",
+                                "iam:Get*",
+                                "rds:DescribeDBInstances",
+                                "rds:ListTagsForResource",
+                                "organizations:ListAccounts",
+                                "ec2:DescribeVolumes",
+                                "ecs:DescribeTaskDefinition",
+                                "ecs:ListTaskDefinitions",
+                                "lambda:GetFunction",
+                                "lambda:ListFunctions",
+                                "lightsail:GetInstances",
+                                "lightsail:GetRegions",
+                                "s3:ListAllMyBuckets",
+                                "cloudtrail:GetTrailStatus",
+                                "cloudtrail:DescribeTrails",
+                                "config:ListDiscoveredResources",
+                                "config:GetDiscoveredResourceCounts",
+                                "eks:DescribeCluster",
+                                "eks:ListClusters",
+                                "ecs:ListContainerInstances",
+                                "ecs:DescribeContainerInstances",
+                                "ecs:DescribeClusters",
+                                "ecs:ListClusters"
```

Comment:
This presumably duplicates https://github.com/expel-io/terraform-aws-cloudtrail/blob/35ef73ad9a2454ebc862706d99086d4cc64bef04/iam.tf#L117-L142 .

* Is there a way to unify them in a single variable?
* If there isn't, can we at least sort them the same way so we can easily compare them, and put a comment in both places about the duplication so people updating it know they have to update both places?

Reply:
I will look into injecting a list into this template so that we can have a single list of truth. Good idea.

Reply:
I attempted a couple of things but didn't get it working within a few hours of attempts so I left a comment & reordered the list to be consistent.